### PR TITLE
bump to dev version

### PR DESCRIPTION
**Pull request by pq**
/cc @devoncarew @mit-mit

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: lints
-description: Official Dart and Flutter lint rules
-version: 0.1.0
+description: Official Dart and Flutter lint rules.
```
Comment (goderbauer) [inline pubspec.yaml:2]: Remove the Flutter part since that's now separate?

Comment (goderbauer) [reply pubspec.yaml:2]: (also in the readme?)

**Review by devoncarew — APPROVED** [review]
lgtm w/ the changes noted by @goderbauer

File: pubspec.yaml (hunk 1 above)
Comment (pq) [reply pubspec.yaml]: Totally.  Good catch.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**